docs/nitpick-exceptions

```diff
@@ -1,2 +1,7 @@
 # photutils.morphology
 py:class photutils.morphology.CompoundModel1
+
+# photutils.detection
+py:class photutils.detection.findstars.StarFinderBase
```

Comment:
@larrybradley - what do you say?
I think we can add these to the public classes (as far as I saw most of the baseclasses are public in astropy core, though only checked a few submodules), and thus the links will also work.